.experimenter.yaml

```diff
@@ -7,6 +7,14 @@ growth-data:
     enabled:
       type: boolean
       description: "If true, the feature is active"
+cookie-banners:
+  description: Features for cookie banners handling.
```

Comment:
I think it should be `cookie banner handling` instead of `banners`. For example in the foxfooding and the docs, that is how they phrase it.